crates/subspace-networking/Cargo.toml

```diff
@@ -42,7 +42,8 @@ features = [
     "tcp-tokio",
     "websocket",
     "yamux",
-    "request-response"
+    "request-response",
+    "identity-hashing-for-peer-id"
```

Comment:
I believe this makes PR non-mergable until we figure out what to do with that feature (feature unification will spill over into other libp2p uses such as in Substrate and will probably make networking in Substrate incompatible with existing network).

Can we merge something almost working without that feature for now and assume all farmers have all pieces?

From there we can iterate and don't have to delay this PR even further. Leave TODO in places that don't work correctly for now.